openwisp_notifications/tests/test_api.py

```diff
@@ -533,7 +533,7 @@ def test_notification_setting_list_api(self):
             self.assertEqual(len(response.data['results']), number_of_settings)
 
         with self.subTest('Test "page_size" query'):
-            page_size = 1
+            page_size = 2
```

Comment:
This test assumes that only one NotificationSetting to be present in the database which was correct when we only had the "default" notification type. We introduced another notification type, so there will be two NotificationSettings in the DB. 

When we set the page_size to 1 in the request, the response would contain only one object per page. Since, there are two objects now in the DB, there will be two pages available for this page size. Thus, the next URL is not empty. 

Read the[ DRF docs on pagination for more details](https://www.django-rest-framework.org/api-guide/pagination/).